<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Space</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="spaceComboBox">
              <property name="toolTip">
               <string/>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Select Space</string>
              </property>
              <item>
               <property name="text">
                <string>Select</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2D</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3D</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Units</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="unitsComboBox">
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Model</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="modelComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="parameter_1_label">
              <property name="text">
               <string>Parameter 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="parameter1LineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="parameter_2_label">
              <property name="text">
               <string>Parameter 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="parameter2LineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QToolButton" name="zoomInToolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/zoom_in.png</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomOutToolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/zoom_out.png</normalon>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="modelGraphicsView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="savePushButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadPushButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
